src/ImageProcessor/Imaging/Helpers/Adjustments.cs

```diff
@@ -329,7 +330,7 @@ private static byte[] GetLinearBytes()
             byte[] ramp = new byte[256];
             for (int x = 0; x < 256; ++x)
             {
-                ramp[x] = (255f * SRGBToLinear(x / 255f)).ToByte();
```

Comment:
Improved the formula but this will never actually work all that well since the inverse LUT should be operating at a minimum of 12 bit precision, something we can never actually get without a custom resampler. That's why it's dropped in V3.